types/src/system/auction/mod.rs

```diff
@@ -399,6 +399,11 @@ pub trait Auction:
 
             non_founder_weights.sort_by(|(_, lhs), (_, rhs)| rhs.cmp(lhs));
 
+            // This assumes that amount of founding validators does not exceed configured validator
+            // slots. For a case where there is exactly N validators and the limit is N, only
+            // founding validators will be the in the winning set. It is advised to set
+            // `validator_slots` larger than amount of founding validators in accounts.toml to
+            // accomodate non-genesis validators.
```

Comment:
That part doesn't apply anymore, does it? We now make sure it's never larger.

Reply:
Yes - now validator slots should be configured to be greater or equal to the number of genesis validator accounts. This comment is for the reader to bring the config validation process.